antarest/launcher/adapters/log_parser.py

```diff
@@ -20,36 +18,27 @@ def update_progress(
         line: str, launch_progress_dto: LaunchProgressDTO
     ) -> bool:
         if "MC-Years : [" in line:
```

Comment:
Plusieurs remarques :

* Je pense que le test avec la regex est suffisant et que l'on n'a pas besoin du `if "MC-Years : [" in line:`, et donc, on n'a pas besoin du `else` avec le warning.
*  Le résultat d'un `re.search`, `re.match` ou `re.fullmatch` est ce qu'on appelle un « Matching Object ». On utilise souvent une variable nommée `mo` au lieu de `regex_result`. On peut faire de même ici…
* Dans une regex, il est préférable d'utiliser `"\s"` pour représenter un espace, car celui-ci aurait pu être ignoré si on avait utilisé le flag `re.VERBOSE`. Je pense qu'on pourrait être un peu plus permissif dans la regex qui analyse la ligne avec "MC-Years :", et accepter un nombre d'espaces variable avec `"\s*"`. À noter qu'il y a une erreur de typo dans le message car, en anglais, il n'y a pas d'espace avant les ":". Si on tient compte de cette erreur de typo, la regex devient : `r"MC-Years\s*:\s*\[\d+\s*..\s*\d+],\s*total\s*:\s*(\d+)"`. Par ailleurs, lorsqu'on a du texte, on peut aussi utiliser le flag `re.IGNORECASE`.